connector/src/main/java/org/geysermc/connector/network/translators/world/block/BlockStateValues.java

```diff
@@ -135,6 +136,13 @@ public static void storeBlockStateValues(Map.Entry<String, JsonNode> entry, int
         if (shulkerDirection != null) {
             BlockStateValues.SHULKERBOX_DIRECTIONS.put(javaBlockState, (byte) shulkerDirection.intValue());
         }
+
+        String key = entry.getKey();
```

Comment:
If you're going to make this a variable, you might as well put it at the top of the file and use it wherever else we call it. I personally would not bother making it a variable.

Reply:
It’s used 4 times below so making it a variable is cleaner